__tests__/endpoint.test.js

```diff
@@ -82,3 +82,40 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test(`when all articles is requested, reponds with all articles in ascending order by creating date and no body`, () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body).toBeSortedBy(body.created_at, { descending: true });
+      });
+  });
+  test(`Each reply must respond with an object which includes author, title, article_id, topic, created_at, votes, article_img_url and comment count `, () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          console.log(body);
+          expect(typeof article.author).toBe("string");
```

Comment:
same here the length check makes sure that your expects in the forEach are being triggered